INSTRUCTION.md

```diff
@@ -0,0 +1,14 @@
+# ToDo App Docker Instructions
+
+## Docker Hub Repository
+You can find the Docker image for the ToDo app here:
+[Docker Hub Repository](https://hub.docker.com/r/zaebalsya13/todoapp)
+
+---
+
+## How to Build and Run the Application
+
+### Step 1: Clone the Repository
+```bash
+git clone https://github.com/zaebalsya1/devops_todolist.git
+cd devops_todolist
```

Comment:
Consider adding instructions for building the Docker image and running the container. For example, you can include commands like `docker build -t todoapp .` and `docker run -p 8080:8080 todoapp` to guide users on how to set up the application locally.